core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -6352,6 +6352,20 @@ private void checkLiteral2(String expression, String expected) {
         .withCalcite().ok(expectedCalciteX);
   }
 
+  /** Test case for
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-5265">[CALCITE-5265]
+   * Select operator' parentheses should be same with Union operator</a>. */
```

Comment:
The comment should also be adapted according to the changes in JIRA title.

Reply:
OK, done.